src/test/ui/traits/issue-104322.rs

```diff
@@ -0,0 +1,80 @@
+// run-pass
```

Comment:
This does not need to be run-pass, just build-pass.

Reply:
```suggestion
// build-pass
```